Makefile

```diff
@@ -9,7 +10,8 @@ watch:
 
 init:
 	brew update
-	brew install hugo
+	brew install hugo pre-commit
```

Comment:
What if someone doesn't want to install `pre-commit` tool?

Reply:
Then that someone can run those commands by hands, nothing prevents they from doing that. 😄

Reply:
Ok